file-formats/sajc/sajc-v1.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$comment": "This file is autogenerated, do not edit manually, see https://github.com/SAP/abap-file-formats for more information.",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "",
+  "title": "SAJC Object Type",
+  "description": "Object type SAJC",
+  "type": "object",
+  "properties": {
+    "formatVersion": {
+      "title": "ABAP File Format Version",
+      "description": "The ABAP file format version",
+      "type": "string",
+      "const": "1"
+    },
+    "header": {
+      "title": "Header",
+      "description": "Header",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "Description of the ABAP object",
+          "type": "string",
+          "maxLength": 120
+        },
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
+          "type": "string",
+          "minLength": 2,
+          "maxLength": 2,
+          "pattern": "^[a-z]+$"
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP language version",
+          "type": "string",
+          "enum": [
+            "standard",
+            "keyUser",
+            "cloudDevelopment"
+          ],
```

Comment:
I tried to find out which different types of ABAP Language Version are possible for SAJC but only found some with "standard". 
@burcuka Do you know more about this or should we ask SAJC colleagues about the language version?

Reply:
I would expect these possible values but Im not sure... maybe we could ask the colleagues

Reply:
I asked the SAJC colleagues.. "standard" and "cloudDevelopment" will be supported as ABAP language versions

Reply:
Then we should either change zif_aff_types_v1=>ty_header_120 to use zif_aff_types_v1=>co_abap_language_version_cloud (and maybe rename it to for example ty_header_120_cloud) or SAJC has to define its own header in its interface